src/app/components/typography/StyledTypography.tsx

```diff
@@ -0,0 +1,11 @@
+import { styled, Typography } from "@mui/material";
+
+export const StyledSectionTypography = styled(Typography)({
+  backgroundImage: "linear-gradient(270deg, hsla(0, 0%, 86.3%, .05), #e6e6e6)",
+  marginTop: "10px",
```

Comment:
perhaps you can make use of "em" to make the app reactive

Reply:
hello @jacob-khoza-symb 
Okay I'll change that thanks.